### PR TITLE
Android: Fixed Export Profile Error

### PR DESCRIPTION
**Pull request by kowalskidev**
Fixed issue: https://github.com/laurent22/joplin/issues/3010

**EISDIR** stands for "Error, Is Directory". This means that NPM is trying to do something to a _file_ but it is a _directory_. In our case app is trying to copy the files from PluginAssets directory but it didn't find any files in that directory so `fsDriver().copy()` is trying to copy the blank directory which causes this error. Now I fixed it by skipping PluginAssets directory while exporting.

@PackElend can you please label me, thank you!

### CODE REVIEW

**Comment by roman-r-m** [comment]
There was another part to that issue where if Joplin has no storage write permissions it fails without asking user for permissions.

**Comment by kowalskidev** [comment]
> 
> 
> There was another part to that issue where if Joplin has no storage write permissions it fails without asking user for permissions.

Yaa I'm aware of that issue, but that's a part of another issue so  we need to create another issue. To discuss further I have already created topic on discourse: https://discourse.joplinapp.org/t/android-issues-in-exporting-profile/7978

File: ReactNativeClient/lib/components/screens/config.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 					const files = await shim.fsDriver().readDirStats(Setting.value('resourceDir'));
 
 					for (const file of files) {
+						if (file.path == 'pluginAssets') continue;
```
Comment (laurent22) [inline ReactNativeClient/lib/components/screens/config.js:121]: A better fix would be to recursively copy all files and folders. I think there's an option on readDirStats for that

Comment (kowalskidev) [reply ReactNativeClient/lib/components/screens/config.js:121]: Okay I'll check them out.

Comment (kowalskidev) [reply ReactNativeClient/lib/components/screens/config.js:121]: I tried passing recursive option `{ recursive: true }` but now it shows different error:
Screenshot_1587212616

@laurent22  If you have any other better way of fixing the issue then please let me know.

Comment (roman-r-m) [reply ReactNativeClient/lib/components/screens/config.js:121]: The path on your screenshot looks suspicious - why is `/data/user/0/net.cozic.joplin/files/` appears twice? Looks like a bug somewhere

Comment (PackElend) [reply ReactNativeClient/lib/components/screens/config.js:121]: @kowalskidev  are you able to debug the origin of the alert

Comment (kowalskidev) [reply ReactNativeClient/lib/components/screens/config.js:121]: I'm working on it

**Comment by kowalskidev** [comment]
@laurent22 Since recursive option didn't work I refactored the code to copy files from sub-directories which also reduced the redundant code now it's working fine. It will export all the directories including PluginAssets > Highlight.js, Mermaid, katex

**Comment by laurent22** [comment]
Ok that looks good, thanks @kowalskidev!

**Pull request merged**